code/game/objects/items/cigs_lighters.dm

```diff
@@ -785,14 +785,14 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	//open flame removed because vapes are a closed system, they wont light anything on fire
 
 	if(super && vapetime > 3)//Time to start puffing those fat vapes, yo.
-		var/datum/effect_system/smoke_spread/chem/s = new
-		s.set_up(reagents, 1, loc, silent=TRUE)
+		var/datum/effect_system/smoke_spread/chem/smoke_machine/s = new
+		s.set_up(reagents, 1, 24, loc)
 		s.start()
 		vapetime = 0
 
 	if((obj_flags & EMAGGED) && vapetime > 3)
-		var/datum/effect_system/smoke_spread/chem/s = new
-		s.set_up(reagents, 4, loc, silent=TRUE)

```

Comment:
removing silence means these will spam admins to hell and back

Reply:
The smoke machine smoke setup doesn't have admin notifications for the actual smoke (to remove spam), so this shouldn't either.

Reply:
???
just leave the silent arg, do you know what it does?